experimental/dds/tree2/src/domains/schemaBuilder.ts

```diff
@@ -55,6 +78,19 @@ export class SchemaBuilder<
 		});
 	}
 
+	public override struct<
```

Comment:
At some point we will want factories for recursive structs I suspect, but I think we can leave that out for now, and worry about that later.